sample.tf

```diff
@@ -10,10 +10,10 @@ terraform {
 
 provider "aws" {
   profile = "default"
-  region  = "us-west-2"
+  region  = "us-west-1"
 }
 
-resource "aws_instance" "app_server" {
+resource "aws_instance" "app_seraver" {
```

Comment:
Issue - Ensure that detailed monitoring is enabled for EC2 instances. 
 category - Logging and Monitoring 
 severity - HIGH